cpu/lpc2387/periph/gpio.c

```diff
@@ -43,6 +43,11 @@ static uint8_t _gpio_isr_map[64]; /* only ports 0+2 can have ISRs */
 
 static void _gpio_configure(gpio_t pin, unsigned rising, unsigned falling);
 
+static inline int __isr_map_entry(unsigned port, unsigned pin)
+{
+    return pin + (port ? 32 : 0);
+}
```

Comment:
Identifiers with two leading underscores are reserved according to the C standard in any context. Additionally, the difference between `__isr_map_entry` and `_isr_map_entry` is hard to see. How about `_isr_map_entry2()`?

Reply:
Ok